test/connectivity/NetworkController/Settings.cs

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace NetworkController
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using Microsoft.Extensions.Configuration;
+
+    class Settings
+    {
+        const string StartAfterPropertyName = "StartAfter";
+        const string DockerUriPropertyName = "DockerUri";
+        const string FrequencyPropertyName = "Frequencies";
+        const string NetworkIdPropertyName = "NetworkId";
+        const string ModePropertyName = "Mode";
+        const string PortsPropertyName = "Ports";
```

Comment:
What ports are these? I don't see them in the config anywhere?
Also, can we pls use a more descriptive name?

Reply:
Not used, I removed it